bundles/org.openhab.binding.emotiva/src/main/resources/OH-INF/config/config.xml

```diff
@@ -45,6 +45,27 @@
 			<default>7100</default>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="activateFrontBar" type="boolean" required="false">
+			<context>feature</context>
```

Comment:
Specific reason this contexst is set to feature while the other two are set to notification?